File: python/7 kyu/404.py
# https://www.codewars.com/kata/5c46ea433dd41b19af1ca3b3/train/python

# Passed

def hex_word_sum(s):
    words = s.split()
    hex_words = [word.upper().replace("O", "0").replace("S", "5") for word in words]
    result = 0
    for hex_word in hex_words:
        try:
            result += int(hex_word, 16)
        except ValueError as e:
            continue

    return result

output = hex_word_sum('CODE')
print(output)

File: nodejs/7 kyu/404.js
// https://www.codewars.com/kata/62cecd4e5487c10028996e04/train/javascript

// Passed

function racePodium(blocks) {
  let secondPlace = Math.ceil(blocks / 3);
  let firstPlace = secondPlace + 1;
  let thirdPlace = blocks - firstPlace - secondPlace;

  if (thirdPlace === 0) {
    thirdPlace++;
    secondPlace--;
  }

  let result = [secondPlace, firstPlace, thirdPlace];
  return result;
}

const output = racePodium(10);
console.log(output);